src/components/views/messages/SenderProfile.tsx

```diff
@@ -113,7 +113,7 @@ export default class SenderProfile extends React.Component<IProps, IState> {
         let member = mxEvent.sender;
         if (SettingsStore.getValue("feature_use_only_current_profiles")) {
             const room = MatrixClientPeg.get().getRoom(mxEvent.getRoomId());
-            if (room) {
+            if (room && member) {
                 member = room.getMember(member.userId);
```

Comment:
```suggestion
            if (room) {
                member = room.getMember(mxEvent.getSender());
```

Reply:
Wouldn't this pass null to getMember though which seems a bit weird?

Reply:
getSender returns the MXID even if there is no sender/member object

Reply:
oh, TIL